3-things-you-must-know-about-midnight-hacks

```diff
@@ -0,0 +1,45 @@
+{  
+  "author": "Kara Dees",
+  "title": "3 Things You Must Know About Midnight Hacks",
+  "thumbnail": "http://imgur.com/a/YATmL",
+  "date": "2017-03-03T02:15:35Z",
+  "tags": ["all"],
+  "published": true
+}
+
+---
+
+##1.  It’s a Phenomenon##
```

Comment:
In Markdown, you only put the hashes (`#`) at the beginning of the line, not the end.

Also, I am not sure what the `1.` is for. It will probably show up in the document, so I would take it off